portfolio/src/main/webapp/uploadscript.js

```diff
@@ -0,0 +1,12 @@
+
+function fetchBlobstoreUrlAndShowForm() {
+  fetch('/blobstore-upload')
+      .then((response) => {
+        return response.text();
+      })
+      .then((imageUploadUrl) => {
+        const messageForm = document.getElementById('my-form');
+        messageForm.action = imageUploadUrl;
+        messageForm.classList.remove('hidden');
+      });
```

Comment:
Some of the indentation appears to be 2 spaces per indentation level mixed with 4 spaces per indentation level, please choose one or the other but do not mix and match